app/assets/stylesheets/govuk-component/_option-select.scss

```diff
@@ -64,58 +64,56 @@
   }
 
   .js-enabled & {
+    padding: 0 0 1px 0;
+
+    .js-selected-counter {
+      @include core-14;
+    }
+    .options-container {
+      border: 5px solid $grey-3;
+    }
 
     /* styles for collapsibleness. .js-collapsible is added by the javascript if the browser is not ie6/7 in which case these don't collapse */
     &.js-collapsible {
-      &:hover {
-        background-color: $grey-2;
-        cursor:pointer;
-      }
+      button {
+        z-index: 100;
+        position: relative  ;
+        background-color: $grey-3;
```

Comment:
This background is redefined below, also you have some extra space before your semi-colon in your position on the line above;